addons/web_editor/static/src/components/media_dialog/file_selector.js

```diff
@@ -256,7 +257,32 @@ export class FileSelector extends Component {
     }
 
     async uploadUrl(url) {
-        await this.uploadService.uploadUrl(url, { resModel: this.props.resModel, resId: this.props.resId }, attachment => this.onUploaded(attachment));
+        if (this.props.uploadUrlAsBinary) {
```

Comment:
I was thinking that we might want e.g. an official PDF from somewhere so remain an actual link, so that the latest version keeps being used. If we agree that this makes no sense, I can remove it.
E.g. "EmergencyPhoneNumbers-June2023.pdf" vs "EmergencyPhoneNumbers-latest.pdf".
But then if we want such a nuance, maybe we will want an option "Link vs Download"... and wanting more options is unlikely.

Reply:
I'd keep it simple and remove this condition so we always download when possible and would not care about the "keep link", in that case they can just create a link without going through the media dialog as a workaround.
What do you think? (It's just my opinion about feature/workaround vs simple code)

Reply:
Sounds good to me, I guess PDE will be fine with it too. I will adapt accordingly.

Reply:
@bso-odoo , you were correct, I agree :) 
If you want a link, you paste a link :D

Reply:
`uploadUrlAsBinary` condition removed.